static/js/reducers/settings.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as api from "../lib/api"
+import { GET, PATCH } from "redux-hammock/constants"
+
+import type { NotificationSetting } from "../flow/settingsTypes"
+
+export const FREQUENCY_IMMEDIATE = "immediate"
+export const FREQUENCY_DAILY = "daily"
+export const FREQUENCY_WEEKLY = "weekly"
+export const FREQUENCY_NEVER = "never"
+
+export const FRONTPAGE_NOTIFICATION = "frontpage"
+
+export const FREQUENCY_CHOICES = [
+  { value: FREQUENCY_NEVER, label: "Never" },
+  { value: FREQUENCY_IMMEDIATE, label: "Immediate" },
```

Comment:
We don't want to let the user select the "Immediate" setting for frontpage digest emails.

Reply:
are we going to want it for other notification types? if so I'll exclude it for just the current UI, if not I'll remove the option entirely

Reply:
Yeah, and in the case of comment notifications we will only want to allow selection of Never / Immediate in the near-term so that makes sense.